cql3/Cql.g

```diff
@@ -1890,7 +1892,9 @@ K_PARTITION:   P A R T I T I O N;
 K_SCYLLA_TIMEUUID_LIST_INDEX: S C Y L L A '_' T I M E U U I D '_' L I S T '_' I N D E X;
 K_SCYLLA_COUNTER_SHARD_LIST: S C Y L L A '_' C O U N T E R '_' S H A R D '_' L I S T; 
 
-K_GROUP:      G R O U P;
+K_GROUP:       G R O U P;
+
+K_LIKE:        L I K E;
```

Comment:
Should `LIKE` be unreserved as well? Don't know, just asking.

Reply:
It should, and it is -- see line 1740.  Same as in the Cassandra grammar.